client/src/Components/Common/Navbar/Menu.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import SVG from "react-inlinesvg";
+
+import { Wrapper, MenuItem } from "./Menu.style.js";
+
+// import ProfileIcon from "./../../../assets/profile-icon.svg";
+
+export default class Menu extends Component {
```

Comment:
`import React, { PureComponent } from "react";`
`export default class Menu extends PureComponent {`

have a read about pure components, there's loads of articles about them

Reply:
ah cool, interesting to use!